### PR TITLE
CORE-1226: added the `total` field to the permanent ID request listing response bodies.

### PR DESCRIPTION
**Pull request by slr71**

### CODE REVIEW

File: project.clj [hunk 1]
```diff
@@ -13,26 +13,26 @@
...
             :url "http://www.iplantcollaborative.org/sites/default/files/iPLANT-LICENSE.txt"}
   :manifest {"Git-Ref" ~(git-ref)}
-  :dependencies [[org.clojure/clojure "1.10.0"]
+  :dependencies [[org.clojure/clojure "1.10.2"]
                  [net.sourceforge.owlapi/owlapi-api "3.5.0"]
                  [net.sourceforge.owlapi/owlapi-apibinding "3.4.10"]
                  [net.sourceforge.owlapi/owlapi-reasoner "3.3"]
                  [me.raynes/fs "1.4.6"]
-                 [cheshire "5.8.1"]
-                 [org.clojure/data.csv "0.1.4"]
```
Comment (ianmcorvidae) [inline project.clj:22]: This is a major version upgrade, I guess we should double-check the interface isn't changed (doesn't look like the rest of the PR touches any CSV stuff)

**Review by ianmcorvidae — APPROVED** [review]
Unrelated comment/request, rest looks good.

Yay, window functions!

File: project.clj (hunk 1 above)
Comment (slr71) [reply project.clj:22]: I checked the release notes before I updated `project.clj`. Here's a link to the diff for good measure.

**Comment by slr71** [comment]
Thanks for the review!

File: project.clj (hunk 1 above)
Comment (ianmcorvidae) [reply project.clj:22]: Cool. It does look like it's basically documentation changes and designating it 1.0, so, seems good. Thanks!

File: src/metadata/persistence/permanent_id_requests.clj [hunk 2]
```diff
@@ -63,11 +59,16 @@
...
   "Lists Permanent ID Requests, filtered and sorted by the given params."
-  [{:keys [statuses] :as params}]
+  [{:keys [statuses] row-limit :limit row-offset :offset :as params}]
   (let [status-clause (when statuses ['in statuses])]
     (map format-request-listing
       (select [(list-permanent-id-requests-query params) :request_list]
-        (where-if-defined {:status status-clause})))))
+        (fields :id :type :target_id :target_type :requested_by :original_path :date_submitted
+                :status :date_updated :updated_by
+                [(raw "count(*) OVER ()") :total])
```
Comment (psarando) [inline src/metadata/persistence/permanent_id_requests.clj:68]: 🎉

**Pull request merged**